### PR TITLE
added omero python recipe

### PR DESCRIPTION
**Pull request by ThomasWollmann**
* [x] I have read the guidelines above.
* [x] This PR adds a new recipe.
* [ ] This PR updates an existing recipe.
* [ ] This PR does something else (explain below).

### CODE REVIEW

File: recipes/python-omero/build.sh [hunk 1]
```diff
@@ -0,0 +1,4 @@
+cd lib/python
+
+CONDA_PYTHON=$(conda info --root)/bin/python
```
Comment (bgruening) [inline recipes/python-omero/build.sh]: Why is this so complicated?
Does a simple `${RECIPE_DIR}/setup.py install` not work?

File: recipes/python-omero/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+  sha1: f94b71ab30d6239e6a4ba2d8b52eca9f262b800a
+  
+build:
+    number: 0
+    skip: True  # [py3k or osx]
+
+requirements:
+  build:
+    - python
+    - conda-build
```
Comment (bgruening) [inline recipes/python-omero/meta.yaml]: This is always available, isn't it?

**Comment by ThomasWollmann** [comment]
This package is still very VIP.

**Comment by druvus** [comment]
Seems that you have a SHA1 mismatch.

**Comment by bgruening** [comment]
I think you also need python as build dependency.

**Pull request merged**